sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/postgres/PostgresFloat8Tests.java

```diff
@@ -0,0 +1,954 @@
+                SELECT sinh('nan'::double);
+                 sinh
+                ------
+                  NaN
+                (1 row)
+                                
+                SELECT cosh('infinity'::double);
+                   cosh  
+                ----------
+                 Infinity
+                (1 row)
+                                
+                SELECT cosh('-infinity'::double);
+                   cosh  
+                ----------
+                 Infinity
+                (1 row)
+                                
+                SELECT cosh('nan'::double);
+                 cosh
+                ------
+                  NaN
+                (1 row)
+                                
+                SELECT tanh('infinity'::double);
+                 tanh
+                ------
+                    1
+                (1 row)
+                                
+                SELECT tanh('-infinity'::double);
+                 tanh
+                ------
+                   -1
+                (1 row)
+                                
+                SELECT tanh('nan'::double);
+                 tanh
+                ------
+                  NaN
+                (1 row)
+                                
+                SELECT asinh('infinity'::double);
+                  asinh  
+                ----------
+                 Infinity
+                (1 row)
+                                
+                SELECT asinh('-infinity'::double);
+                   asinh  
+                -----------
+                 -Infinity
+                (1 row)
+                                
+                SELECT asinh('nan'::double);
+                 asinh
+                -------
+                   NaN
+                (1 row)
+                
+                SELECT '32767.4'::double::int2;
+                 int2
+                -------
+                 32767
+                (1 row)
+                
+                SELECT '-32768.4'::double::int2;
+                  int2
+                --------
+                 -32768
+                (1 row)
+                
+                SELECT '-32767.6'::double::int2;
+                  int2
+                --------
+                  -32767
+                (1 row)
+                
+                SELECT '2147483647.4'::double::int4;
+                    int4
+                ------------
+                 2147483647
+                (1 row)
+                
+                
+                SELECT '2147483647.6'::double::int4;
+                    int4
+                ------------
+                 2147483647
+                (1 row)
+                
+                SELECT '-2147483648.4'::double::int4;
+                    int4
+                -------------
+                 -2147483648
+                (1 row)
+                """
+        );
+    }
+
+/*
+                -- check edge cases for exp --> we don't support exp yet
```

Comment:
I have added EXP to the list of functions to implement https://github.com/feldera/feldera/issues/1265